src/blueprints/docs/templates/components/topic.vue

```diff
@@ -31,7 +31,11 @@ export default {
       { property: 'og:site_name', content: this.$docs.title }
     ]
 
-    if (this.meta.description) {
+    if (this.$press.docs.image) {
+      meta.push({ property: 'og:image', content: this.$press.docs.image })
```

Comment:
```suggestion
      meta.push({ property: 'og:image', content: this.$docs.image })
```
`this.$docs` is the ref to the docs blueprint config for the current pressId, see the docs mixin